### PR TITLE
Qdrant (Independent Publisher)

### PR DESCRIPTION
**Pull request by Anush008**
---

Adds a connector to include support for Qdrant - https://qdrant.tech/

I've tested with the test suites and templates.

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Screenshot 2024-09-26 at 11 05 21 AM
Screenshot 2024-09-26 at 11 18 28 AM

### CODE REVIEW

File: independent-publisher-connectors/Qdrant/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          "tooltip": "Provider your hostname",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#090E1A",
```
Comment (vmanoharas) [inline independent-publisher-connectors/Qdrant/apiProperties.json]: Please change the iconBrandColor as per the template in the below link. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json#L4

**Review by vmanoharas — COMMENTED** [review]
Hi @Anush008, 

Please resolve review comments.

**Review by vmanoharas — APPROVED** [review]
Dear Partner

I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by Anush008** [comment]
Hey @vmanoharas.
I received the following mail from conndep@microsoft.com.

> [Action Required] Connector Ownership Acknowledgment for qdrant

> If the email is not rendered, please check that your mail client supports the [Adaptive Card feature](https://docs.microsoft.com/en-us/outlook/actionable-messages/#outlook-version-requirements-for-actionable-messages).
As a workaround you can use the [Outlook on the web for Office 365](https://outlook.office.com/).

I've tried to open this on Outlook as well. I don't see anything else in the mail body.

Do you happen to know anything about this?

**Comment by vmanoharas** [comment]
> Hey @vmanoharas. I received the following mail from [conndep@microsoft.com](mailto:conndep@microsoft.com).
> 
> > [Action Required] Connector Ownership Acknowledgment for qdrant
> 
> > If the email is not rendered, please check that your mail client supports the [Adaptive Card feature](https://docs.microsoft.com/en-us/outlook/actionable-messages/#outlook-version-requirements-for-actionable-messages).
> > As a workaround you can use the [Outlook on the web for Office 365](https://outlook.office.com/).
> 
> I've tried to open this on Outlook as well. I don't see anything else in the mail body.
> 
> Do you happen to know anything about this?

Hello @Anush008, 

Kindly ignore this email as of now.

**Comment by vmanoharas** [comment]
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short survey - https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**